src/commands/batch.py

```diff
@@ -91,7 +91,7 @@ def view(batch_id, node):
         if job == None: raise ClickException('No job found')
         table_data = [
             ['Date', job.created_date],
-            ['Batch', job.batch.id],
+            ['Batch ID', job.batch.id],
```

Comment:
Keep it as `Batch ID` here. This is a vertical table so the additional space is available. As this views a single `Job`, it could get confused with `Job ID`.